tasks/sep/021-stage-1-4/README.md

```diff
@@ -0,0 +1,37 @@
+# Stage 1.4 Upgrade
+
+Status: DRAFT, NOT READY TO SIGN
+
+## Objective
+
+Registers a new `FaultDisputeGame` with the `DisputeGameFactory` that is backed by the [`asterisc`][asterisc] FPVM and
+the [`kona`][kona] fault proof program.
+
+This upgrades the Fault Proof contracts in the
+[op-contracts/v1.9.0-rc.3](https://github.com/ethereum-optimism/optimism/tree/op-contracts/v1.9.0-rc.3) release.
+
+## Pre-deployments
+
+- `RISCV` - `0x7f9a34da8d240bbe039e9f7b497c9bc500f6acf5`
```

Comment:
How can I found these addresses predeployed? 
For Mainnet, this one would be inside the gov proposal?